src/js/components/utils/useForm.tsx

```diff
@@ -50,6 +50,14 @@ export default ({
     }
     setInputs({ ...inputs, [e.target.name]: value });
   };
+  const handleSubmitFromRef = (data: any) => {
+    dispatch(
+      updateObject({
+        objectType: OBJECT_TYPES.PROJECT,
+        data,
+      }),
+    );
+  };
```

Comment:
Is the goal of this to use `updateObject` instead of `createObject`? If so, I'd recommend adding a new arg to `useForm` that allows the user to choose between the two, instead of creating a new handler like this. Perhaps a `model` arg, and if that's present then it calls `updateObject` instead of `createObject`?

Reply:
Yes!  that's the plan to add a prop to identify the type of object.  This is just here to see how the call to update should look.

Reply:
Yeah, something similar to this. I expect we'll have to make some changes to `updateObject` as well -- it should likely accept a `hasForm` arg (like `createObject` does).